addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -426,6 +426,100 @@ QUnit.test('chatter should become enabled when creation done', async function (a
     );
 });
 
+QUnit.test('readmore/less span do not duplicate', async function (assert) {
```

Comment:
`span` is technical detail, more like "button link".
Also name of test doesn't not relate to what it does / asserts. I think you should avoid naming tests in a negative way, and instead tell what it should do.